app/controllers/api/v1/open_library_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class OpenLibraryController < ApiController
+      def show
+        books = OpenLibrary.new(params[:isbn]).books
+        render json: books
```

Comment:
What happens if the service does not respond with a 200 or does not respond at all (timeout)?